projects/pulsar_adc_pmdz/zed/system_project.tcl

```diff
@@ -0,0 +1,46 @@
+###############################################################################
+## Copyright (C) 2016-2023 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+source ../../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+## The following HDL projects supports all the devices of EVAL-AD40XX-FMCZ:
+##
+##  AD4000/AD4001/AD4002/AD4003/AD4004/AD4005/AD4006/AD4007/AD4008/AD4010/AD4011/AD4020
+##
+## and also the EVAL-ADAQ400x eval board (with PMOD):
+##
+##  ADAQ4003
+##
+## NOTE: Make sure that you set up your required ADC resolution and sampling rate
```

Comment:
We no longer need to set the sample rate because it can be change by the software.

Reply:
I think that we can add a comment in pulsar_adc_pmdz_bd.tcl script that describes the data_width values based on converter resolution.

Reply:
Done.